src/plugins/janus_sip.c

```diff
@@ -4510,6 +4511,8 @@ static void *janus_sip_handler(void *data) {
 			record_peer_audio = peer_audio ? json_is_true(peer_audio) : FALSE;
 			json_t *peer_video = json_object_get(root, "peer_video");
 			record_peer_video = peer_video ? json_is_true(peer_video) : FALSE;
+			json_t *peer_pli = json_object_get(root, "send_peer_pli");
```

Comment:
Since you added a new argument, this needs to be added to the automated validation ("is this a boolean?"), otherwise people may send a different type causing issues. You can check where "peer_video" and "peer_audio" are validated and add your new property there too: this is currently happening in `recording_parameters` in the code.